rdkit/VLib/NodeLib/UnitTestNodeLib.py

```diff
@@ -0,0 +1,94 @@
+# $Id$
```

Comment:
The two UnitTest files run the doc tests as well. I now removed them from the test_list.py files and added the UnitTestNodeLib and UnitTestVLib files instead.